components/quest/QuestsList.tsx

```diff
@@ -0,0 +1,102 @@
+import { FlatList, RefreshControl, StyleSheet, View } from 'react-native';
+import { useGetQuestsQuery } from './api';
+import QuestCard from './QuestCard';
+import ShimmerPlaceholder from 'react-native-shimmer-placeholder';
+import FetchError from '../common/FetchError';
+import { LinearGradient } from 'expo-linear-gradient';
+import { Ionicons } from '@expo/vector-icons';
+import Colors from '../../theme/colors';
+import React, { useCallback, useState } from 'react';
+import QuestCountdownTimer from "./QuestCountdownTimer";
+import {useFocusEffect} from "@react-navigation/native";
+
+const QuestsList = () => {
+  const { data: quests, isLoading, isError, refetch } = useGetQuestsQuery();
```

Comment:
when we do these with `isLoading` it probably won't show the skeleton on subsequent fetches, we can use `isFetching` instead.